New version: DiplodocusCollisions v0.0.2

JuliaRegistrator opened the pull request:
- Registering package: DiplodocusCollisions
- Repository: https://github.com/cneverett/DiplodocusCollisions.jl
- Created by: @cneverett
- Version: v0.0.2
- Commit: 8b7e2d4a76ac4ed6e12220e8a7e9cf1cd3df4b30
- Reviewed by: @cneverett
- Reference: https://github.com/cneverett/DiplodocusCollisions.jl/commit/8b7e2d4a76ac4ed6e12220e8a7e9cf1cd3df4b30#commitcomment-164435931
- Description: Julia Evaluation of the Collision Integrals for Discrete Particle Interactions
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
## Breaking changes

- There are no breaking changes
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd65cf52cc4d037a5de603ae51d7e8f2cae554420e0f347e1d4ac5711f799b559384f5f75375737519272e9db4e29f9945f7d086ed068404129fc3e9b9b551da21980f6c828a957c4420111f4b28ade0c4566fb36375f620b134667789463b6115f6746a4fc47ab2df3f60d3689d31827df3aa0ad257742893675316a7e0af358887de8df62e3dd0e76f56592c14fb33add755455296cbc616862229ee52a57004a5d6da96b76e0f8f664280b7393d39c785848d1c5c2858e854918ab473fc9bf40 -->

cneverett commented:
@JuliaRegistrator register

Release notes:

## Breaking changes

- There are no breaking changes

JuliaRegistrator committed:
New version: DiplodocusCollisions v0.0.2

UUID: 32d0d58a-4da0-4f8e-8b7e-c2816ac6e5cb
Repo: https://github.com/cneverett/DiplodocusCollisions.jl.git
Tree: df17d8feb49d11b9aefd5a30e736f4adbf28f543

Registrator tree SHA: 50f504d641745716a5b3eabaf681d3a4937d2ae3
